tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -167,6 +178,53 @@
 # It is quite clear that the MRI surfaces (head, brain) are not well aligned
 # to the head digitization points (dots).
 #
+# Compute the transform
+# ---------------------
+# Let's step through the process that the transform is accomplishing.
+
+
+def plot_alignment(points):
+    renderer_kwargs = dict(bgcolor='w', smooth_shading=False)
+    renderer = mne.viz.backends.renderer._get_renderer(
+        size=(800, 400), **renderer_kwargs)
+    seghead_rr, seghead_tri = mne.read_surface(os.path.join(
+        subjects_dir, 'sample', 'surf', 'lh.seghead'))
+    renderer.mesh(*seghead_rr.T, triangles=seghead_tri, color=(0.7,) * 3,
+                  opacity=0.2)
+    for point in points.copy():
+        point -= 128  # RAS zero centered
+        renderer.sphere(center=(point[0], point[2], point[1]),
+                        color='r', scale=5)
+    view_kwargs = dict(elevation=90, azimuth=0)
+    view_kwargs['focalpoint'] = (0., 0., 0.)
+    mne.viz.set_3d_view(figure=renderer.figure, distance=1000, **view_kwargs)
+    renderer.show()
+
+
+###############################################################################
+# Get landmarks in head space from the DigMontage stored in raw.
+# -------------------------------------------------------------
+# Adjust units by * 1e3 m -> mm.
+head_space = np.array([dig['r'] for dig in
+                       raw.info['dig']], dtype=float) * 1e3
+plot_alignment(head_space)
+
+
+###############################################################################
+# Transform digitization points into MRI space (T2)
+# -------------------------------------------------
+mri_space = mne.transforms.apply_trans(trans, head_space, move=True)
```

Comment:
`trans` is in `m` so you need to adjust the translation component:
```
trans_mm = trans.copy()
trans_mm[:3, 3] *= 1e3
```
then use it instead of `trans`. Or alternatively use `trans` on the head points in meters, then multipy by `1e3`